tests/unit/UI/Alert/DialogBox.spec.js

```diff
@@ -0,0 +1,29 @@
+import { mount } from "@vue/test-utils";
+import DialogBox from "@/components/UI/Alert/DialogBox";
+
+describe("DialogBox.vue", () => {
+  it("should render with default values", () => {
+    const wrapper = mount(DialogBox);
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("renders title correctly", () => {
+    const title = "Dialog Title";
+    const wrapper = mount(DialogBox, {
+      props: {
+        title,
+      },
+    });
+    expect(wrapper.find("p").text()).toBe(title);
+  });
+
+  it("renders description correctly", () => {
+    const description = "Dialog Title";
+    const wrapper = mount(DialogBox, {
+      props: {
+        description,
+      },
+    });
+    expect(wrapper.find("p").text()).toBe(description);
+  });
```

Comment:
shouldn't we ideally run one more with both of them being passed to ensure that both the correct `p` tags are being set?

Reply:
Ideally, there's no harm. More the merrier!

I see that there can be more test cases added to this component and I think let's plan and work on them at once?